src/current/_includes/common/license-expiration-messages.md

```diff
@@ -0,0 +1,9 @@
+Depending on the status of your cluster's [license]({% link {{ page.version.version }}/licensing-faqs.md %}), the following messages may be displayed in the DB Console:
+
+- If the license will expire soon, the message is: _This cluster will require a license key by ${DATE} or the cluster will be throttled._
+- If the license is no longer valid, the message is: _Your license key expired on ${DATE} and the cluster was throttled. Please add a license key to continue using this cluster._
+- If the cluster is required to send telemetry but has not been sending it, the message is: _Telemetry has not been received from some nodes in this cluster since ${DATE}. These nodes will be throttled on ${DATE} unless telemetry is received._
```

Comment:
There is a separate message displayed after the throttling occurs due to telemetry requirements: 
"Telemetry has not been received from some nodes in this cluster since <date>. These nodes were throttled on July <date>. Learn more"

Reply:
OK, added another bullet and ordered them together as follows:

- If the cluster is required to send telemetry but has not been sending it, and is _not yet_ being throttled, the message is: _Telemetry has not been received from some nodes in this cluster since ${DATE}. These nodes will be throttled on ${DATE} unless telemetry is received._
- If the cluster is required to send telemetry but has not been sending it, and is _currently_ being throttled, the message is:  _Telemetry has not been received from some nodes in this cluster since ${DATE1}. These nodes were throttled on ${DATE2}._

Reply:
@jaiayu ok with the updates i made here to resolve this?